app/routes/joblistings/components/JoblistingList.css

```diff
@@ -63,43 +60,36 @@
   height: 80px;
   min-width: 120px;
   object-fit: contain;
+  margin-right: 20px;
 }
 
 .listItem {
+  display: flex;
   flex: 1;
   justify-content: space-between;
+  font-size: 14px;
 
-  @media (max-width: 900px) {
+  @media (--mobile-device) {
     flex-direction: column;
   }
 }
 
 .joblistingItemTitle {
-  color: var(--color-black);
-  font-weight: 550;
-  margin: 0;
-  text-transform: uppercase;
+  color: var(--color-dark-gray-1);
+  font-weight: 600;
   font-size: 15px;
```

Comment:
Perhaps increase this a bit to compensate for making the text lower case? Again I find a little too small